src/org/omegat/core/dictionaries/StarDict.java

```diff
@@ -199,6 +202,8 @@ public StarDictDict(File ifoFile) throws Exception {
                 mem.close();
                 idx.close();
                 bais.close();
+                MapDoubleArray<Object> result = new MapDoubleArray(loaded);
+                loaded = null;
```

Comment:
There is no reason to set null to this variable when it is a local variable that will go out of scope immediately afterwards.